_core/smart/functions.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: core
+permalink: /:collection/:path.html
+---
+# Using functions
```

Comment:
This section describes more than functions -- it seems like "Defining Functions and Data Maps" would fit more closely.

Reply:
Done.